src/en/space-station-14/round-flow/proposals/time-agent.md

```diff
@@ -0,0 +1,83 @@
+* **Benevolent**: The future self uses 1-2 brute-healing items on the targeted entity if the entity has higher brute damage than burn damage, 1-2 burn-healing items if burn damage is higher than brute damage, or uses the equivalent of Help Intent if the target has no burn damage and no brute damage. If there are no entities on the targeted tile, the future self will target the closest person to the tile in a 7 tile radius that isn't the present self. If that still yields no valid results, the future self guards the targeted location for 7 seconds. If, within 7 seconds, a valid entity is seen by it, the future self runs to them to do the things listed above. If a wall/window was targeted, the future self attempts to fix it with the appropriate tools & materials required. If a disposal unit is targeted, the future self attempts to activate it to send anything in it through.
+
+#### Assist the Past:
+Teleports the agent into a projection of the past - a temporary, empty dimension with featureless figures representing any mobs that were nearby at the time - for however many seconds the unclosed loop had lasted, where all performed actions had already happened before. The damage and other effects that the agent's future self experienced when summoned in reality will be dealt during the assistance in the projection. The center of the temporary dimension’s floor glows the most, and is where **Summon Future Assistance** was targeted originally.
+
+#### Temporal Loop Memories:
+The agent recalls the actions they must perform with **Assist the Past** to close the time loops created by **Summon Future Assistance**.
+
+#### United Synchronization:
+Can be cycled through freely. Multiplied uses of abilities incur multiplied amounts of possibilities, as is expected.
+* **None**: The default.
+* **Past Only**: Using **Summon Past Assistance** uses the ability twice if there are 2 or more **Assist the Future** loops currently prepared for closure.
+* **Past & Future**: Using **Summon Past Assistance** uses the ability as many times as there is **Assist the Future** loops currently prepared for closure. If the amount of prepared **Assist the Future** loops is less than 3, **Summon Future Assistance** is used however many times needed to bring the total number of summoned doppelgangers up to 3. Using **Summon Future Assistance** uses the ability thrice.
+* **Future Only**: Using **Summon Future Assistance** uses the ability twice.

```

Comment:
i dont understand why this needs to exist, couldnt you just use the ability multiple times anyway

Reply:
...True. The abilities don't have cooldowns anyhow. We axe this, then?